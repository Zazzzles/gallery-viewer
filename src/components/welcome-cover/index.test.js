import React from 'react';
import renderer from 'react-test-renderer';

import WelcomeCover from './index';

it('renders correctly', () => {
  const tree = renderer.create(<WelcomeCover />).toJSON();
  expect(tree).toMatchSnapshot();
});
