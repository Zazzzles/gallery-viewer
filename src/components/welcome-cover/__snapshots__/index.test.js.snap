// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Container"
>
  <div
    className="Inner"
  >
    <span
      className="Title"
    >
      Welcome!
    </span>
    <img
      className="KeysImage"
      src="keys.png"
    />
    <div
      className="TextContainer"
    >
      <span
        className="TextHeading"
      >
        Pro tip!
      </span>
      <span
        className="Text"
      >
        Use your directional arrows to naviate the interface
      </span>
    </div>
    <div
      className="ContinueContainer"
    >
      <span
        className="ContinueText"
      >
        Enter to continue
      </span>
      <img
        className="EnterIcon"
        src="enter.png"
      />
    </div>
  </div>
</div>
`;
